### ISSUE TITLE
Is there conversational memory, or just document retrieval? (Question)

### ISSUE THREAD

**Issue by soccerwithag** [gustavz/DataChad#5]
Since ConversationBufferMemory is not used, wondering if/how there is conversational memory. Sorry if dumb question. Appreciate this work. Thanks

**Comment by gustavz** [comment]
Yes we store conversation in a streamlit session state variable `st.session_state["chat_history"]` and pass it to the chain on each call.
